scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -62,14 +71,41 @@ fi
 bundle_id=${1:-$HOSTNAME}
 dest_dir=${2:-$DEFAULT_DEST_DIR}
 
-while getopts ":b:t:c:s:" flag
-do
-    case "${flag}" in
-          b) bundle_id="$OPTARG";;
-          t) dest_dir="$OPTARG";;
-          c) config_url="$OPTARG";;
-          s) sevice="$OPTARG";;
-    esac
+duration=P5d
+size_limit=0
+services=All
+stacktrace=false
+coredumps=false
+binlogs=true
+TEMP=$( getopt -o c: -l duration:,size_limit:,services:,binlogs:,coredumps:,stacktrace: \
+                  -n "$PROG" -- "$@")
+eval set -- "$TEMP"
+while true; do
+     case "$1" in
+         -c)config_url=$(echo $2 | sed -e 's/,/\n/g'); 
+            shift 2 ;;
+         --duration)
+            duration=$(echo $2 | sed -e 's/,/\n/g') ;
+            shift 2 ;;
+         --size_limit)
+            size_limit=$(echo $2 | sed -e 's/,/\n/g') ;
+            shift 2 ;;
+         --services)
+            services=$(echo $2 | sed -e 's/,/\n/g') ;
```

Comment:
If someone specifies pipe separated "|" multiple service, will we be captured all those here in $services ?

Reply:
> 

for service we have no op that is why captured all those here in $services